Simplenote/src/main/java/com/automattic/simplenote/TagDialogFragment.java

```diff
@@ -33,52 +35,42 @@
     private AlertDialog mDialogEditTag;
     private Bucket<Note> mBucketNote;
     private Bucket<Tag> mBucketTag;
+    private Button mButtonNegative;
     private Button mButtonPositive;
     private String mTagOld;
     private Tag mTag;
     private TextInputEditText mEditTextTag;
     private TextInputLayout mEditTextLayout;
+    private TextView mMessage;
+    private View.OnClickListener mClickListenerNegativeConflict;
+    private View.OnClickListener mClickListenerNegativeRename;
+    private View.OnClickListener mClickListenerPositiveConflict;
+    private View.OnClickListener mClickListenerPositiveRename;
```

Comment:
Question (❓  ): Is the Hungarian notation still used in this repo? I have seen elsewhere that it is a mixture of using it so I am not sure. If it has been decided to not use that anymore then consider dropping the `m` prefix from all those new fields.

Reply:
So, for new fields should we use the Hungarian notation or not? I am asking that because I assumed we shouldn't and I haven't included it on the fields I created so far.

Reply:
Since the codebase is 89% Java and 0% Kotlin, I would use Hungarian notation even though it's not a strictly required code style in Simplenote Android.  Hungarian is the accepted notation for Java and camel case is the accepted notation for Kotlin.  In this file, all variables use Hungarian notation.  So, keep that notation makes sense to me.

Reply:
I see how you are thinking about it, thanks for sharing Tyler! 🌟 

But, Hungarian notation is language-independent, it doesn't have to be Java vs Kotlin.

When I argue about that, I found this [article](https://jakewharton.com/just-say-no-to-hungarian-notation/) very enlighting. I can add very little to that.